ccd-definition/CaseField.json

```diff
@@ -41,6 +41,14 @@
     "_Definition":  "The details of the first applicant on the case",
     "_Category":  "COMMON"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "claimantLitigationFriend",
```

Comment:
can we change this to applicant1LitigationFriend?

Reply:
Yep, will do tomorrow

Reply:
Ta